catalog/spatial/registry/registry-api-impl/src/main/java/org/codice/ddf/registry/api/impl/RegistryStoreImpl.java

```diff
@@ -197,8 +198,8 @@ public SourceResponse query(QueryRequest request) throws UnsupportedQueryExcepti
 
         //This is a registry store so only allow registry requests through
         if (!filterAdapter.adapt(request.getQuery(),
-                new TagsFilterDelegate(Collections.singleton(RegistryConstants.REGISTRY_TAG),
-                        true))) {
+                new TagsFilterDelegate(Sets.newHashSet(RegistryConstants.REGISTRY_TAG,
```

Comment:
As this was immutable before - `Collections.singleton()` returns an immutable set - we should switch to `ImmutableSet` here.